lms/djangoapps/philu_overrides/tests/test_context_processor.py

```diff
@@ -0,0 +1,590 @@
+@mock.patch('lms.djangoapps.philu_overrides.context_processor.is_org_oef_prompt_available', mock.Mock())
+@mock.patch('lms.djangoapps.philu_overrides.context_processor.get_org_oef_update_prompt', mock.Mock())
+@mock.patch('lms.djangoapps.philu_overrides.context_processor.is_org_detail_prompt_available')
+@mock.patch('lms.djangoapps.philu_overrides.context_processor.get_org_metric_update_prompt')
+def test_overlay_message_oef_url_failure(
+    mock_get_org_metric_update_prompt,
+    mock_is_org_detail_prompt_available,
+    mock_is_org_detail_platform_overlay_available,
+    request_object  # pylint: disable=redefined-outer-name
+):
+    """
+    Test that get_global_alert_messages returns overlay_message = None when request.path contains '/oef/dashboard'
+    """
+
+    request = request_object
+    request.path = request.path + '/oef/dashboard'
+
+    mock_get_org_metric_update_prompt.return_value = True
+    mock_is_org_detail_prompt_available.return_value = True
+    mock_is_org_detail_platform_overlay_available.return_value = True
+
+    expected_overlay_message = None
+    actual_overlay_message = get_global_alert_messages(request).get('overlay_message')
+
+    assert expected_overlay_message == actual_overlay_message
+
+
+@mock.patch('lms.djangoapps.philu_overrides.context_processor.is_org_detail_platform_overlay_available')
+@mock.patch('lms.djangoapps.philu_overrides.context_processor.is_org_oef_prompt_available', mock.Mock())
+@mock.patch('lms.djangoapps.philu_overrides.context_processor.get_org_oef_update_prompt', mock.Mock())
+@mock.patch('lms.djangoapps.philu_overrides.context_processor.is_org_detail_prompt_available')
+@mock.patch('lms.djangoapps.philu_overrides.context_processor.get_org_metric_update_prompt')
+def test_overlay_message_org_url_failure(
+    mock_get_org_metric_update_prompt,
+    mock_is_org_detail_prompt_available,
+    mock_is_org_detail_platform_overlay_available,
+    request_object  # pylint: disable=redefined-outer-name
+):
+    """
+    Test that get_global_alert_messages returns overlay_message = None when request.path contains '/oef/dashboard'
+    """
+
+    request = request_object
+    request.path = request.path + '/organization/details/'
+
+    mock_get_org_metric_update_prompt.return_value = True
+    mock_is_org_detail_prompt_available.return_value = True
+    mock_is_org_detail_platform_overlay_available.return_value = True
+
+    expected_overlay_message = None
+    actual_overlay_message = get_global_alert_messages(request).get('overlay_message')
+
+    assert expected_overlay_message == actual_overlay_message
+
+# Tests for other context processors
+
+
+def test_add_nodebb_endpoint():
+    request = 'request'
+    expected_result = {'nodebb_endpoint': settings.NODEBB_ENDPOINT}
+    actual_result = add_nodebb_endpoint(request)
+    assert expected_result == actual_result
+
+
+def test_get_cdn_link():
+    request = 'request'
```

Comment:
Pylint's stats attached in description of PR, 10/10. Pylint does not enforce here to add a docstring, maybe because of the small size of the function.